app/config.js

```diff
@@ -34,7 +34,8 @@
     loggingURL: "",
     metricsURL: ""
   };
-
+  // Additional support only for Chinese now: zh-CN
+  window.OPENSHIFT_LANG = "en";
```

Comment:
Should this be based on the user's language settings in the browser instead of a server setting? It should be available in `navigator.languages` in most browsers.

https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/languages

Reply:
To add to @spadgett 's point, config.js in the real console server is generated at startup and then cached in the server's memory so we don't have to re-evaluate the internal template every time someone hits it, this means we can't include content in this file that changes per user.

Reply:
@jwboyer Agree.